src/util.ts

```diff
@@ -1,27 +1,33 @@
-/* eslint-disable @typescript-eslint/no-var-requires */
-/* eslint-disable @typescript-eslint/explicit-function-return-type */
-const crypto = require('crypto');
-const database = require('./database');
-const logger = require('./logger');
-const grpc = require('nice-grpc');
-const config = require('../config.json');
-const { SETTINGS } = require('./models/settings');
-const { CONTENT } = require('./models/content');
-const { NOTIFICATION } = require('./models/notifications');
-const { COMMUNITY } = require('./models/communities');
-const { AccountDefinition } = require('@pretendonetwork/grpc/account/account_service');
-const { FriendsDefinition } = require('@pretendonetwork/grpc/friends/friends_service');
-const { APIDefinition } = require('@pretendonetwork/grpc/api/api_service');
-const translations = require('./translations');
-const HashMap = require('hashmap');
-const TGA = require('tga');
-const pako = require('pako');
-const PNG = require('pngjs').PNG;
-const bmp = require('bmp-js');
-const aws = require('aws-sdk');
-const crc32 = require('crc/crc32');
-const communityMap = new HashMap();
-const userMap = new HashMap();
+import aws from 'aws-sdk';
+import bmp from 'bmp-js';
+import crc32 from 'crc/crc32';
+import crypto from 'crypto';
+import HashMap from 'hashmap';
+import grpc from 'nice-grpc';
```

Comment:
```suggestion
import { createChannel, createClient, Metadata } from 'nice-grpc';
```
Similar to the redis issue, the default export for grpc doesn't include the functions we need, so call them out specifically